mvrx/src/main/kotlin/com/airbnb/mvrx/MvRxViewModelFactory.kt

```diff
@@ -94,18 +94,24 @@ data class ActivityViewModelContext(
 /**
  * The [ViewModelContext] for a ViewModel created with a
  * fragment scope (`val viewModel by fragmentViewModel<MyViewModel>`).
+ *
+ * The [ownerProvider] and [savedStateRegistry] default to the Fragment as the provider, if you need to use an alternative
+ * source for the [ViewModelStoreOwner] and/or [SavedStateRegistry]; such in the case of JetPack Navigation Component destinations with ViewModels,
+ * then you can provided them via this context.
  */
 data class FragmentViewModelContext(
     override val activity: FragmentActivity,
     override val args: Any?,
     /**
      * The fragment owner of the ViewModel.
      */
-    val fragment: Fragment
+    val fragment: Fragment,
+    private val ownerProvider: ViewModelStoreOwner = fragment,
+    private val savedStateRegistryProvider: SavedStateRegistry = fragment.savedStateRegistry
```

Comment:
Can we  remove the member properties and just make the the constructor params non-private?

I find it confusing that the params are named `ownerProvider` but the value is the owner itself, not a provider of the owners.

Reply:
Ah, yes, thank you. Yer, it is a bit redundant. Updated.